lib/logitech_receiver/base_usb.py

```diff
@@ -128,7 +134,16 @@
 LIGHTSPEED_RECEIVER_C53f = _lightspeed_receiver(0xc53f)
 LIGHTSPEED_RECEIVER_C53d = _lightspeed_receiver(0xc53d)
 
-del _DRIVER, _unifying_receiver, _nano_receiver, _lenovo_receiver, _lightspeed_receiver
+# Wired devices
```

Comment:
I think it would be more useful if there were descriptions of what these devices are.